PhotoTaggingGramplet/PhotoTaggingGramplet.py

```diff
@@ -228,13 +227,13 @@ def build_gui(self):
 
         button_panel = Gtk.HBox()
 
-        self.button_index = Gtk.ToolButton(Gtk.STOCK_INDEX)
-        self.button_add = Gtk.ToolButton(Gtk.STOCK_ADD)
-        self.button_del = Gtk.ToolButton(Gtk.STOCK_REMOVE)
-        self.button_clear = Gtk.ToolButton(Gtk.STOCK_CLEAR)
-        self.button_edit = Gtk.ToolButton(Gtk.STOCK_EDIT)
-        self.button_zoom_in = Gtk.ToolButton(Gtk.STOCK_ZOOM_IN)
-        self.button_zoom_out = Gtk.ToolButton(Gtk.STOCK_ZOOM_OUT)
+        self.button_index = Gtk.ToolButton(stock_id=Gtk.STOCK_INDEX)
+        self.button_add = Gtk.ToolButton(stock_id=Gtk.STOCK_ADD)
+        self.button_del = Gtk.ToolButton(stock_id=Gtk.STOCK_REMOVE)
+        self.button_clear = Gtk.ToolButton(stock_id=Gtk.STOCK_CLEAR)
+        self.button_edit = Gtk.ToolButton(stock_id=Gtk.STOCK_EDIT)
+        self.button_zoom_in = Gtk.ToolButton(stock_id=Gtk.STOCK_ZOOM_IN)
+        self.button_zoom_out = Gtk.ToolButton(stock_id=Gtk.STOCK_ZOOM_OUT)
```

Comment:
You should not be using this method of locating stock icons.  It has been deprecated for a very long time now.  I quote `Deprecated since version 3.10: Use named icon “preferences-system” or the label “_Preferences”.`
This applies to all the stock icons.

Reply:
Thanks for the feedback and please be patient with me.   

This method of locating icons is not something I did - it's from the addon-source I built off of.   (https://github.com/gramps-project/addons-source/commit/841958c528496ee9534e547d14258033189ee3a8?branch=841958c528496ee9534e547d14258033189ee3a8&diff=split) .  It's more than likely I have the PR on the wrong branch.  

Regardless, I'm happy to clear up the deprecation but I'm not familiar with Gtk.  I trying to find online resources to explain how the code needs to be changed.  I am now lost and confused.   I found a document that tells me STOCK_CLEAR needs to become edit-clear - progress.   I just can't figure out what to do with this new information.  

So this 
`self.button_clear = Gtk.ToolButton(stock_id=Gtk.STOCK_CLEAR)   `

world become...
`self.button_clear = Gtk.ToolButton('edit-clear') `

Would you be able to point me in a good direction?
Thanks,
Bruce.